core-ui/public/assets/version.yaml

```diff
@@ -0,0 +1 @@
+version: dev
```

Comment:
what does that do? I think this works like the file is mounted by a config map on prod? But probably not yet. You can take the info if this is development or not without a requst by checking the global `process.env.NODE_ENV`. 

I'm not sure about this approach but talked with Piotr and agreed that as long as we are mounting the files in Luigi, we should take them from Luigi? Maybe this mount can be already moved to React's assets

Reply:
This is a temporary solution until we remove Luigi. More info [here](https://github.com/kyma-project/busola/issues/1924#issuecomment-1286885130).